src/notation/view/percussionpanel/percussionpanelmodel.cpp

```diff
@@ -224,8 +224,7 @@ void PercussionPanelModel::finishEditing(bool discardChanges)
     }
 
     // Return if nothing changed after edit...
-    if (inst->drumset() && updatedDrumset
-        && *inst->drumset() == *updatedDrumset) {
+    if (inst->drumset() && *inst->drumset() == updatedDrumset) {
```

Comment:
At this point we're already certain that `inst->drumset()` is not nullptr, because of the check above, right?